src/web/modules/transactions/transactions.http.ts

```diff
@@ -77,3 +79,44 @@ export async function show(req: Request, res: Response, next: NextFunction): Pro
     next(error)
   }
 }
+
+export async function statistics(req: Request, res: Response, next: NextFunction): Promise<void> {
+  try {
+    let account
+    const accountId = req.query.account
+
+    if (req.query.account) {
+      account = await AdminUsers.gatewayAccountServices(accountId)
+    }
+
+    const periodKeyMap: {[key:string]: string} = {
+      today: 'day',
+      week: 'week',
+      month: 'month'
+    }
+    const selectedPeriod: any = req.query.period || 'today'
+    const momentKey = periodKeyMap[selectedPeriod]
+
+    const fromDate: string = moment().utc().startOf(momentKey).format()
+    const toDate: string = moment().utc().endOf(momentKey).format()
+  
+    const paymentsByState = await Ledger.getPaymentsByState(accountId, fromDate, toDate)
+    const paymentStatistics = await Ledger.paymentStatistics(accountId, fromDate, toDate)
+
+    const results = {
+      payments: paymentStatistics.count,
+      gross: paymentStatistics.gross_amount,
+      success: paymentsByState.success,
+      error: paymentsByState.error,
+      in_progress: paymentsByState.started
```

Comment:
I think in progress is made up of a number of states https://github.com/alphagov/pay-toolbox/blob/master/src/lib/pay-request/api_utils/ledger.js#L25